reproducibility_project/src/engines/cassandra/project.py

```diff
@@ -376,6 +376,71 @@ def run_cassandra(job):
         )
 
 
+@Project.operation
+@Project.pre(cassandra_complete)
+@Project.post(lambda job: "mean_energy_box1" in job.document)
+def statistics(job):
+    """Compute statistical quantities for each job."""
+    import ele
+    from unyt import angstrom, g, mole
+
+    proplist = [
+        "energy_total",
+        "volume",
+        "nmols",
+        "pressure",
+        "density",
+        "mass_density",
+        "energy_angle",
+        "energy_dihedral",
+        "energy_intravdw",
+        "energy_intraq",
+        "energy_inter",
+        "energy_intervdw",
+        "energy_lrc",
+        "energy_interq",
+        "energy_recip",
+        "energy_self",
+        "enthalpy",
+    ]
+
+    NA = 6.032e23
+
+    C = ele.element_from_symbol("C")
+    H = ele.element_from_symbol("H")
+
+    mw = C.mass + H.mass * 4
+    mw *= g / mole
+
+    if job.sp.ensemble == "GEMC-NVT":
+
+        box1 = "prod.out.box1.prp"
+        box2 = "prod.out.box2.prp"
+
+        data_box1 = pd.read_table(
+            job.fn(box1), skiprows=3, sep="\s+", names=proplist
+        )
+        data_box2 = pd.read_table(
+            job.fn(box2), skiprows=3, sep="\s+", names=proplist
+        )
+
+        job.document.mean_energy_box1 = data_box1["energy_total"].mean()
+        job.document.mean_energy_box2 = data_box2["energy_total"].mean()
+
+    else:
+
+        box1 = "prod.out.prp"
+
+        data_box1 = pd.read_table(
+            job.fn(box1), skiprows=3, sep="\s+", names=proplist
+        )
+
+        job.document.mean_energy_box1 = data_box1["energy_total"].mean()
+        job.document.mean_density_box1 = (
+            data_box1["density"].mean() / NA * mole / (angstrom ** 3)
+        ).to("mole/cm**3") * mw
```

Comment:
You can convert this to a `unyt_quantity` if you really want to put a `unyt_quantity `in the job document, but you have to change your `unyt` import.  I just did it without unyt so that this suggested change can work on its own.  You can also remove `NA` and the MW stuff from the function.
```suggestion
        job.document.mean_density_box1 = 
            data_box1["mass_density"].mean() * 0.001
```